cmd/cli/job/describe.go

```diff
@@ -60,13 +64,127 @@ func (o *DescribeOptions) run(cmd *cobra.Command, args []string) {
 	ctx := cmd.Context()
 	jobID := args[0]
 	response, err := util.GetAPIClientV2().Jobs().Get(ctx, &apimodels.GetJobRequest{
-		JobID: jobID,
+		JobID:   jobID,
+		Include: "executions",
 	})
+
 	if err != nil {
 		util.Fatal(cmd, fmt.Errorf("could not get job %s: %w", jobID, err), 1)
 	}
 
-	if err = output.OutputOneNonTabular(cmd, o.OutputOpts, response.Job); err != nil {
-		util.Fatal(cmd, fmt.Errorf("failed to write job %s: %w", jobID, err), 1)
+	if o.OutputOpts.Format != "" {
+		if err = output.OutputOneNonTabular(cmd, o.OutputOpts, response); err != nil {
+			util.Fatal(cmd, fmt.Errorf("failed to write job %s: %w", jobID, err), 1)
+		}
+		return
+	}
+
+	job := response.Job
+	var executions []*models.Execution
+	if response.Executions != nil {
+		// TODO: #520 rename Executions.Executions to Executions.Items
+		executions = response.Executions.Executions
+	}
+
+	o.printHeaderData(cmd, job)
+	o.printExecutionsSummary(cmd, executions, jobID)
+	o.printExecutions(cmd, executions, jobID)
+	o.printOutputs(cmd, executions)
```

Comment:
Might it be best if these returned errors so we could decorate them with "failed to display job" and have the errors returned have more useful info?

Reply:
Each of these methods are calling `util.Fatal` internally with a decorated error message. `util.Fatal` will call `os.Exit` and the following print commands won't be called. I can change this if you thing it is a better approach

Reply:
Actually your suggestion is better. Done in https://github.com/bacalhau-project/bacalhau/pull/3459/commits/9aca5d0d3f3bc7b2909970bee3190ea33d15d3b1